### PR TITLE
Adding Liquibase integration

### PR DESCRIPTION
**Pull request by r2liquibase**
**Requirements**

- [x] I have followed the repository's [pull request submission guidelines](../blob/main/README.md#submitting-pull-requests)
- [x] I have updated my user documentation
- [x] I have updated my developer documentation (my integration README)
- [x] I have submitted the [Become a Partner](https://launchdarkly.com/partners/) form

**Intended availability**

If you're adding a new integration, let us know if you are a community member or a partner.

If you are a partner, your integration will be available to all customers.

If you're part of the community and submitting an integration for your own purposes or internal teams, specify your company name so that we can verify your identity and enable the integration for your account.

**Related issues**

Provide links to any issues in this repository or elsewhere relating to this pull request.

**Describe the solution you've provided**

If you're adding a new integration, provide some context into how LaunchDarkly and the third-party service should interact.

If you're modifying an existing integration, provide a clear and concise description of what you expect to happen.

**Describe alternatives you've considered**

If you're modifying an existing integration, provide a clear and concise description of any alternative solutions or features you've considered.

**Provide a request preview**

Include the output from running `npm run preview <INTEGRATION_NAME>`.

**Additional context**

Add any other context about the pull request here.

### CODE REVIEW

File: integrations/liquibase/manifest.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Liquibase",
+  "version": "1.0.0",
+  "overview": "Liquibase helps developers bring progressive deliver and DevOps to the database",
+  "description": "Liquibase Launch Darkly Extension gives users the ability to use Launch Darkly to define and control when database changes are applied. The extensions allow for feature flags to be used as Preconditions in changelogs and changesets.",
```
Comment (InTheCloudDan) [inline integrations/liquibase/manifest.json:5]:
```suggestion
  "description": "Liquibase LaunchDarkly Extension gives users the ability to use LaunchDarkly to define and control when database changes are applied. The extensions allow for feature flags to be used as Preconditions in changelogs and changesets.",
```

File: integrations/liquibase/manifest.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "supportEmail": "support@liquibase.com",
+  "links": {
+    "site": "https://liquibase.com",
+    "launchdarklyDocs": "https://docs.liquibase.com/feature-flags/readmes/launchdarkly%20feature%20flags.html",
+    "privacyPolicy": "https://www.liquibase.com/privacy-policy"
+  },
+  "categories": ["developer-tools", "automation"],
+  "icons": {
+    "square": "assets/images/Liquibase_logo_vertical_RGB.svg",
+    "horizontal": "assets/images/Liquibase_logo_horizontal_RGB.svg"
```
Comment (InTheCloudDan) [inline integrations/liquibase/manifest.json:16]: @r2liquibase please update the filenames to match the filenames in the `assets/images` directory or rename the files there so they match what is included in the manifest.

**Review by InTheCloudDan — CHANGES_REQUESTED** [review]
@r2liquibase thank you for the PR! There's only a few minor changes required. I'm looking into the linting issue now you shouldn't need to handle that one in this case. If that changes I'll let you know.

File: integrations/liquibase/manifest.json (hunk 2 above)
Comment (r2liquibase) [reply integrations/liquibase/manifest.json]: Absolutely.

**Comment by r2liquibase** [comment]
Thanks, @InTheCloudDan . Changes made.

**Review by InTheCloudDan — APPROVED** [review]

**Pull request merged**